check_script.py

```diff
@@ -228,14 +229,10 @@ def check_script(fname, *args, **kw):
     else:
         pylint_path = "{path}"
 
-    if not kw.get("P3", False):
-        pycmd = find_pylint(2)
-        rc_file = "tmp_pylint.rc"
-    else:
-        pycmd = find_pylint(3)
-        rc_file = "tmp_pylint3.rc"
+    pycmd = find_pylint(3)
```

Comment:
I assume this is the change that causes this code to be tested under pylint3 but it's not clear, perhaps a comment here would help?  Will this change cause all SConstruct files to be tested with pylint3 or just the ones in this repo?

It looks like the find_pylint(version) function could be simplified now as well.

Do we still have test code that uses python2?

Reply:
This would change it to python3 for scons_local.  Test code will have to update.  I could add a P2 option.  But it will only affect daos and cart when they update scons_local